sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -1684,112 +1183,6 @@ private void ValidateUpdatedDataFeedWithOptionalMembersSet(DataFeed dataFeed, st
             Assert.That(dataFeed.IngestionSettings.DataSourceRequestConcurrency, Is.EqualTo(6));
         }
 
-        private void ValidateAzureApplicationInsightsDataSource(AzureApplicationInsightsDataFeedSource dataSource)
```

Comment:
Removing the specific data feed source validation methods since we don't need them anymore. They all have been moved to the same `ValidateDataSource` method (added in the previous PR).